### PR TITLE
Upgraded node version to 18

### PR DESCRIPTION
**Pull request by SansCaar**

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: '16'  # Use Node.js 16.x 
+          node-version: '18'  # Use Node.js 16.x
```
Comment (christikaes) [inline .github/workflows/tests.yaml:18]: can you update this comment too pls 😅

**Review by christikaes — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yaml (hunk 1 above)
Comment (SansCaar) [reply .github/workflows/tests.yaml:18]: Sure!

**Comment by SansCaar** [comment]
I don't know how this got closed

**Comment by SansCaar** [comment]
Reopening. didn't know how it got closed

**Comment by SansCaar** [comment]
image
Image of test

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -15,19 +15,16 @@ jobs:
...
         with:
-          node-version: '16'  # Use Node.js 16.x 
+          node-version: '18'  # Use Node.js 18.x 
 
       - name: Install pnpm
-        run: npm install -g pnpm
-      
-      - name: Changing Directory
-        run: cd nepalingo-web
+        run: npm -C nepalingo-web install -g pnpm
```
Comment (christikaes) [inline .github/workflows/tests.yaml:21]: Let's use this pnpm action: https://pnpm.io/continuous-integration#github-actions

**Review by christikaes — CHANGES_REQUESTED** [review]

**Review by christikaes — APPROVED** [review]
Looks like the build is passing now - thanks for fixing this!

**Comment by binamkayastha** [comment]
Breaking my only author merges rule here again, and just merging it in to unblock other prs and get this to done

**Pull request merged**